### ISSUE TITLE
feathers generate app failed with @feathersjs/transport-commons@^4.5.5

### ISSUE THREAD

**Issue by ruojianll** [feathersjs/cli#232]
### Steps to reproduce

Run `feathers generate app`.

TS
app-name
[empty]
src
npm
REST + SOCKET.io
Jest
Y
Username+password
users
Sequelize
MySql
default connection string

### Expected behavior

Create a new project.

### Actual behavior
Got error:
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @feathersjs/transport-commons@^4.5.5.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of '@feathersjs/authentication'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/lilei/.npm/_logs/2020-07-20T11_20_33_018Z-debug.log
```

**Comment by daffl** [comment]
That version [is available on npm](https://www.npmjs.com/package/@feathersjs/transport-commons) and I am not able to reproduce on a fresh installation. Try running the command again with the latest Node installed [via nvm](https://github.com/nvm-sh/nvm).

**Comment by ruojianll** [comment]
@daffl I tried yarn and latest version 4.5.5 has not found too, then I chosen 4.5.3.

**Comment by daffl** [comment]
Since nobody else seems to be running into this, it's likely something in your local setup (app caches, proxy settings etc.). I also just published version 4.5.7 of all packages which may also solve this.

**Comment by UtmostCreator** [comment]
In my case I was needed to do the following steps:

1. `npm cache clean --force` - this step is outdated, so it could be skipped (most probably)
2. `npm cache verify`
3. npm uninstall feathers
4. npm uninstall -g feathers

After these steps the application was generated successfully.

<details>
  <summary>My Log</summary>

  ```
feathers g app
? Do you want to use JavaScript or TypeScript? JavaScript
? Project name server
? Description
? What folder should the source files live in? src
? Which package manager are you using (has to be installed globally)? npm
? What type of API are you making? REST, Realtime via Socket.io
? Which testing framework do you prefer? Mocha + assert
? This app uses authentication No
? Which coding style do you want to use? ESLint
   create package.json
   create config\default.json
   create public\favicon.ico
   create public\index.html
   create .editorconfig
   create src\app.hooks.js
   create src\channels.js
   create src\index.js
   create src\logger.js
   create src\middleware\index.js
   create src\services\index.js
   create .gitignore
   create README.md
   create src\app.js
   create test\app.test.js
   create .eslintrc.json
   create config\production.json
   create config\test.json
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated @types/socket.io-parser@3.0.0: This is a stub types definition. socket.io-parser provides its own type definitions, so you do not need this installed.

added 155 packages, removed 212 packages, and audited 162 packages in 24s

9 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

added 212 packages, and audited 374 packages in 14s

45 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

> server@0.0.0 lint
> eslint src/. test/. --config .eslintrc.json --fix "--fix"
  ```
</details>
